lib/folio_record.rb

```diff
@@ -17,10 +17,61 @@ def instance_id
     record.dig('externalIdsHolder', 'instanceId')
   end
 
+  def sirsi_holdings
+    @sirsi_holdings ||= items.map do |item|
```

Comment:
this is kind of weird. at this point I almost feel like we should make a `FolioHolding` that does nothing other than inherit from `SirsiHolding`, just so we stop ending up with odd things like this that we might forget to change the names of